bundles/org.openhab.binding.miele/src/main/java/org/openhab/binding/miele/internal/handler/DishwasherChannelSelector.java

```diff
@@ -73,20 +73,8 @@ public State getState(String s, @Nullable DeviceMetaData dmd, MieleTranslationPr
         }
     },
     PROGRAM_PHASE(RAW_PHASE_PROPERTY_NAME, PHASE_CHANNEL_ID, DecimalType.class, false, false),
-    START_TIME("startTime", "start", DateTimeType.class, false, false) {
-        @Override
-        public State getState(String s, @Nullable DeviceMetaData dmd, MieleTranslationProvider translationProvider) {
-            Date date = new Date();
-            SimpleDateFormat dateFormatter = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
-            dateFormatter.setTimeZone(TimeZone.getTimeZone("GMT+0"));
-            try {
-                date.setTime(Long.valueOf(s) * 60000);
-            } catch (Exception e) {
-                date.setTime(0);
-            }
-            return getState(dateFormatter.format(date));
-        }
-    },
+    START_TIME("", START_CHANNEL_ID, DateTimeType.class, false, false),
+    END_TIME("", END_CHANNEL_ID, DateTimeType.class, false, false),
```

Comment:
Just to be sure, it is expected to have the first parameter as an empty string ?
And not START_TIME_PROPERTY_NAME  for the first one ?

Reply:
Yes, it i s overridden by the handler, so channel selector is not used.